### PR TITLE
Fixed all the broken links in README.md

### LINKED ISSUE #146: All the links in README.md file are broken
# What / Why

> The links are all broken.

## Where

* In the README.md file

### PR DESCRIPTION
**Pull request by marsonya**
All the links in the README.md file were pointing to incorrect urls.

### CODE REVIEW

**Comment by marsonya** [comment]
The 'what-is-npm' redirects to https://github.com/npm/cli. I couldn't find any file suitable for that link in this repo.

File: README.md [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+<li><a href="/npm-license.md">npm License</a></li>
+<li><a href="/privacy.md">Privacy Policy</a></li>
+<li><a href="/unpublish.md">Unpublish Policy</a></li>
+<li><a href="/receiving-reports.md">Receiving Abuse Reports</a></li>
+<li><a href="/dmca.md">Copyright and DMCA Policy</a></li>
+<li><a href="/trademark.md">Trademark Policy</a></li>
+<li><a href="/security.md">Security</a></li>
+<li><a href="/crawlers.md">Replication and web crawler policy</a></li>
+<li><a href="/domains.md">Our official list of domains</a></li>
+<li><a href="https://github.com/npm/cli#npm1----a-javascript-package-manager">What is npm?</a></li>
```
Comment (MylesBorins) [inline README.md:18]:
```suggestions
```

I think we can just remove this link since it no longer exists

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by ethomson** [comment]
These pages are built for our _website_ and checked in to this repository for visibility and traceability.  But that means that the links are formatted for our website at https://www.npmjs.com/policies.  Changing them so that they're useful in the context of this repository - at the moment - breaks the links on our website.

Obviously it would be nice if they were consistent and usable in both places, but the tooling does not do this at the moment.  I'm reverting this to fix the website.

**Pull request merged**